content/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -251,6 +251,118 @@ log entries for `v1` and none for `v2`:
     127.0.0.1 - - [07/Mar/2018:19:26:44 +0000] "GET /headers HTTP/1.1" 200 361 "-" "curl/7.35.0"
     {{< /text >}}
 
+1. If you want to take a look inside the traffic, run the following commands on another console and send the traffic:
```

Comment:
`If you want to examine traffic internals, run the following commands on another console:`.

Reply:
the part about "send the traffic:" is vague - how  is it sent?

Reply:
Thank you for your reviewing. I will modify the description as you wrote.
`If you want to examine traffic internals, run the following commands on another console:`

"Send the traffic" means to run a command to send the sleep pod traffic on the former console like the previous step.
I think the readers would understand the context without the sentence if they read the steps from the start.

Reply:
I fixed in [dff2916](https://github.com/istio/istio.io/pull/3267/commits/dff2916128000757c978fd6367359b2854ef97db).